mappings/dnd5e.mapping

```diff
@@ -44,22 +44,22 @@
     { "pdf": "Deception", "foundry": @data.skills.dec.total },
     { "pdf": "History", "foundry": @data.skills.his.total },
     { "pdf": "Wpn Name", "foundry": @items.filter(i => i.type === 'weapon' && i.data.data.equipped && i.hasAttack && i.hasDamage)[0]?.name || "" },
-    { "pdf": "Wpn1 AtkBonus", "foundry": "" },
-    { "pdf": "Wpn1 Damage", "foundry": @items.filter(i => i.type === 'weapon' && i.data.data.equipped && i.hasAttack && i.hasDamage)[0]?.data?.data?.damage?.parts[0][0] || "" },
+    { "pdf": "Wpn1 AtkBonus", "foundry": @items.filter(i => i.type === 'weapon' && i.data.data.equipped && i.hasAttack && i.hasDamage)[0]?.labels.toHit || "" },
+    { "pdf": "Wpn1 Damage", "foundry": (function() { let dda = Array.from(actor.items.filter(i => i.type === 'weapon' && i.data.data.equipped && i.hasAttack && i.hasDamage))[0]?.labels.derivedDamage; return dda[0].formula + (dda?.length > 1 ? " *" : "")})() },
```

Comment:
The Weapon Damage fields are labelled `DAMAGE/TYPE` on the PDF, so perhaps it would be appropriate to also add the damage type to the output?

More importantly, you should handle for when the result of this expression is undefined by adding optional chaining in your return statement: `return dda?.[0]` and outputting an empty string if it's undefined: `?? ""`.

Reply:
Added dmg type and fixed null/undef handling